kura/org.eclipse.kura.api/META-INF/MANIFEST.MF

```diff
@@ -65,7 +65,7 @@ Export-Package: org.eclipse.kura;version="1.6.0",
  org.eclipse.kura.status;version="1.0.2",
  org.eclipse.kura.system;version="1.6.0",
  org.eclipse.kura.type;version="1.1.0",
- org.eclipse.kura.usb;version="1.3.0",
+ org.eclipse.kura.usb;version="2.0.0",
```

Comment:
Why these changes if we are just adding a class in a package?

Reply:
The new package should also be exported, I think.

Reply:
because the baseline check plugin gave me an error preventing the build.

Reply:
Do you have the same error with develop?

Reply:
in develop i haven't the same problem, i try to revert and see if the error persist.